sdks/java/core/src/main/java/org/apache/beam/sdk/options/ExecutorOptions.java

```diff
@@ -19,10 +19,7 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.util.concurrent.MoreExecutors;
-import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.util.concurrent.ThreadFactoryBuilder;
+import org.apache.beam.sdk.util.UnboundedScheduledExecutorService;
 
 public interface ExecutorOptions extends PipelineOptions {
 
```

Comment:
comment and `@Description` needs to be updated since the core pool size is irrelevant now.